configs/rust/targets/aarch64-bottlerocket-linux-gnu.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Just for my own understanding, how were these files generated?

Reply:
I generated them using the `update-rust.sh` script.